### ISSUE TITLE
what's the basis for setting the learning rate

### ISSUE THREAD

**Issue by jkkj1630** [facebookresearch/wav2letter#581]
Hello,I am a novice,but i used train at kaldi tdnn-f and cnn-lstm-ctc at tensorflow,but they learn rate seem a small number,like 0.001,0.0001 or 0.01,i saw in the wav2letter librispeech example learn rate is big number,like 0.5 ,0.1,is there any difference between them?what's the basis for setting the wav2letter learning rate?if i try train total time is 10 minute data,how much learn rate more suitable,and 100 minute?100hour,10000hour?
my network config file is https://github.com/facebookresearch/wav2letter/blob/master/tutorials/1-librispeech_clean/network.arch

**Comment by vineelpratap** [comment]
I'm haven't looked into those frameworks in detail, but in wav2letter - we divide the learning rate by the total batchsize automatically. For example, if you are running an experiment with 4 GPUs and batchsize of 8, the learning rate will be divided by 32. 

> if i try train total time is 10 minute data,how much learn rate more suitable,and 100 minute?100hour,10000hour?

I would start with a grid search which worked best for 10 minutes and see if it works. Otherwise, you can do a grid search around it and see which works best.

**Comment by jkkj1630** [comment]
thank for you answer,i try it!
